### DISCUSSION TITLE
Installation in IPv6 environment?

### DISCUSSION THREAD

**Discussion by josh-haas** [Security-Onion-Solutions/securityonion#5007] [Unsupported Versions]
New environment - I've been unable to get a heavy node to complete syncing to manager. Environment is supposed to be IPv6 for management connections. Manager IPv6 address comes up during setup on the heavy node, but setup invariably fails. I keep finding bits of log everywhere on both nodes that look like they're not working correctly over v6. Is so-setup not fully IPv6 ready?

**Comment by feynmanliang** [comment]
I am having IPv6 issues as well. In particular, the SANs on certs are hard-coded to IPv4 addresses so I am seeing `LogstashClient.cpp:662 SSL handshake with Logstash server at securityonion:5055 encountered an error: (null)` errors being generated from elastic agent.

**EDIT**: I fixed my issue by using a hard-coded IPv4 addresses for logstash rather than relying on DHCP DNS (which was resolving an IPv6).
